### PR TITLE
unit validation table. payload schema changes

### PR DESCRIPTION
**Pull request by mmisztal1980**
## Payload refactoring
- readings -> data
- all measurements are now pascal-case

## Unit validation
- added table
- added min-max values for temp
- added min-max values for atmospheric pressure
- added min-values for mass units
- added min-max values for air humidity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,26 +251,35 @@ Payload shall be an array of at least one **observation**.
...
-- `PM2_5` – PM2.5 (unit: **µg/m³**)
-- `SO2` – Sulfur Dioxide (unit: **µg/m³**)
-- `TEMP` – Temperature (unit: **Celsius**)
-- `HUM` – Humidity (unit: **%**)
-- `PRES` - Atmospheric Pressure (unit: **hPa**)
-
-All float values.
+**(!)** Unless a given value has no unit, all values are referenced using the SI unit system.
+**(!)** All JSON property names should be **pascal-case**
+|General Symbol|JSON Object Property|Name            |Unit   |Min Value|Max Value|Notes|
```
Comment (MaciejGad) [inline README.md]: there is still an error with displaying this table, it should be formatted like this:

First Header | Second Header
------------ | -------------
Content from cell 1 | Content from cell 2
Content in the first column | Content in the second column

File: README.md [hunk 2]
```diff
@@ -251,26 +251,37 @@ Payload shall be an array of at least one **observation**.
...
+O3            |o3                  |Ozone           |[µg/m³]|0.00     |?        |     
+PM10          |pm10                |PM10            |[µg/m³]|0.00     |?        |     
+PM2.5         |pm2_5               |PM2.5           |[µg/m³]|0.00     |?        |     
+SO2           |so2                 |Sulfur Dioxide  |[µg/m³]|0.00     |?        |     
+T             |temp                |Temperature     |[degC] |-273.15  |60.00    |(1)  
+Hum           |hum                 |Humidity        |[%]    |0.00     |100.00   |     
+P             |press               |Atm Pressure    |[hPa]  |870.00   |1083.80  |(2)  
+
+
+**(1)** Min recorded temp = **0[degK]** = **-273.15[degC]**. Max recorded temp = **56.7 [degC]** (1922 - Death Valley, USA)
```
Comment (MaciejGad) [inline README.md]: `The lowest natural temperature ever directly recorded at ground level on Earth is −89.2 °C (−128.6 °F; 184.0 K), which was at the Soviet Vostok Station in Antarctica, on July 21, 1983.` and `Low Temperature Laboratory recorded a record low temperature of 100 pK, or 1.0 × 10−10 K in 1999.` (https://en.wikipedia.org/wiki/Lowest_temperature_recorded_on_Earth)

so sentence `Min recorded temp = **0[degK]** = **-273.15[degC]**` is not exactly precise, especially if you present  **56.7 [degC]**  as max recorded temp.

Comment (mmisztal1980) [inline README.md]: Makes sense. I'll fix it

Comment (mmisztal1980) [inline README.md]: fixed

Comment (mmisztal1980) [inline README.md]: @MaciejGad how do I flag this as resolved? is this something that you need to do on your end?

**Comment by mmisztal1980** [comment]
Requested changes have been applied. unit validation values have been modified, to compensate for the eventuality of measuring new record-breaking values.

**Pull request merged**